types/milesight/EM410-RDL/default.schema.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$id": "https://akenza.io/milesight/EM410-RDL/default.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "processingType": "uplink_decoder",
+  "topic": "default",
+  "title": "Default",
+  "properties": {
+    "temperature": {
+      "$ref": "https://raw.githubusercontent.com/akenza-io/device-type-library/main/data-models/environment/schema.json#/$defs/temperature/celsius"
+    },
+    "distance": {
+      "$ref": "https://raw.githubusercontent.com/akenza-io/device-type-library/main/data-models/distance/schema.json#/$defs/distance/cm"
+    },
+    "distanceMutation": {
```

Comment:
what does distanceMutation mean? distanceDelta?

Reply:
I do think so too, but its not defined properly in the documentation.